website/_i18n/en/user-guides/pactus-testnet/index.md

```diff
@@ -0,0 +1,96 @@
+---
+version: 8
+---
+
+## Preface
+
+Pactus Blockchain has a Test network for developers to test their software which is using Pactus, users
```

Comment:
Pactus Blockchain has introduced a Test network called Phoenix for developers and users to test their software utilizing Pactus technology. This guide will demonstrate how to connect and interact with Phoenix using official Pactus software designed for the main network.